dali/kernels/imgproc/flip_cpu.h

```diff
@@ -90,18 +99,19 @@ class DLL_PUBLIC FlipCPU {
  public:
   DLL_PUBLIC FlipCPU() = default;
 
-  DLL_PUBLIC KernelRequirements Setup(KernelContext &context, const InTensorCPU<Type, 4> &in) {
+  DLL_PUBLIC KernelRequirements Setup(KernelContext &context, const InTensorCPU<Type, 5> &in) {
```

Comment:
Can you put some meaningful name to this magic 5?

Reply:
Done